modules/portworx/scripts/remove-portworx.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+set -u
+set -o pipefail
+. ../../vars
+
+vPXStrgClstrName="px-cluster";
+"${kbCtl}"  --kubeconfig="${vKubeConfig}" patch   storagecluster "${vPXStrgClstrName}" --namespace portworx -p '{"spec":{"deleteStrategy":{"type":"UninstallAndWipe"}}}' --type=merge
+"${kbCtl}"  --kubeconfig="${vKubeConfig}" delete  StorageCluster "${vPXStrgClstrName}" --namespace portworx &
+sleep 10
+echo "Wating for portworx cleanup proces to finish. It will take about 5 minutes..."
+sleep 300;
```

Comment:
A while loop with a 5 min timeout to check if objects are deleted will be useful here instead of waiting always 5 min